participants/DBC-AI-Distributed-Cloud-Storage.md

```diff
@@ -0,0 +1,66 @@
+# DBC-AI-Distributed-Cloud-Storage
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name:
+    - *DBC-AI-Distributed-Cloud-Storage*
+
+- Team Slack Handles:
+    - *deepbrainfeng@gmail.com*
+
+## (1) Brief Project Description
+
+*DBC AI Distribute Cloud Storage is a core component of deep brain chain, DeepBrain Chain is a network of distributed artificial intelligence work force, there are a large number of users in the use of GPU, artificial intelligence AI training, we hope to be able to get the user's data can be stored in the GPU machine not only, also can backup stored in filecoin network, so that users can download data at any time to any GPU machine to calculate.*
+
+## (2) Link to Project UI
+
+*https://github.com/DeepBrainChain/DBC-AI-Distributed-Cloud-Storage.git*
+
+## (3) What does your application/UI do?
+
+*This is a web version of cloud disk, our artificial intelligence users can easily upload data to the network disk, and then users can download the data of the network disk in any gpu machine.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*We didn't use the data.*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will not collect data. These data belong to our AI users, and we do not have the right to use these data. These data are the data of artificial intelligence developers themselves. We have not done any audit on the data, nor can we know whether the data contains sensitive information. However, these data are all the data of users, and no one else has the right to access the data.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*We hope to upload about 1TB of data. At present, there are more than 15000 artificial intelligence developers and more than 1000 GPUs. The actual user data is much larger than 1TB.*
+
+
+## (7) How are you structuring the data?
+
+*We will upload the user's original data directly, and the specific data format is determined by the user himself.*
```

Comment:
Can you provide an example of the data structure?

Reply:
There are many data formats of ARTIFICIAL intelligence, including text and pictures. What do you mean by data format 
example